snowflake/datadog_checks/snowflake/check.py

```diff
@@ -82,6 +84,25 @@ def renew_token(self):
         with open(self._config.token_path, 'rb', encoding="UTF-8") as f:
             self._config.token = f.read()
 
+    def read_key(self):
```

Comment:
The token function is named `renew_token` and this is named `read_key`, could we harmonize these names? Additionally, why do we read the token before the connection, and read the key file while making it?

Reply:
I think it's better to not store the key in the integration ; I'll do a bit of refactor in a next PR to do the same with the token alongside updating the validators

Reply:
Sounds good!